.github/workflows/ci.yml

```diff
@@ -38,3 +34,27 @@ jobs:
         uses: codecov/codecov-action@v1
         with:
           token: ${{secrets.CODECOV_TOKEN}}
+
+  publish:
+    name: publish docker image
+    runs-on: ubuntu-latest
+    env:
+      DOCKER_USERNAME: "${{ secrets.DOCKER_USERNAME }}"
+      DOCKER_PASSWORD: "${{ secrets.DOCKER_PASSWORD }}"
+    if: github.ref == 'refs/heads/master'
+    steps:
+      - uses: actions/checkout@v2
+      - name: Fetch repository tags (for git describe to work)
+        run: git fetch --prune --unshallow
+
+      # Build and push docker image
+      # Make sure that the DOCKER_USERNAME and DOCKER_PASSWORD secret variables are set
+      - name: Export env vars
+        run: |
+          export RELEASE=$(git describe --tags)
+          if [ -z "$RELEASE" ]; then echo "Could not determine release name"; exit 1; fi
+          echo "export DOCKER_TAGS='studentenportal/web:latest studentenportal/web:${RELEASE}'" >> .ci-env
```

Comment:
Did you see [set-env](https://help.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable)? I think this could be solved cleaner with it.

Reply:
Oh, that's nice! TIL

Reply:
I'm not sure what the reason is, but due to some kind of processing GH does, the env var is not being passed in properly:

https://github.com/studentenportal/web/pull/254/checks?check_run_id=539902259

I'll revert the changes, the set-env trick is nice but it's also a bit intransparent what happens and the syntax is not clearly defined. (For example, how should spaces be escaped? What about multi-line variables?)